packages/core/src/transport/eventBridge.ts

```diff
@@ -26,10 +26,16 @@ export function getEventBridge<T, E>() {
   }
 }
 
-export function canUseEventBridge(): boolean {
+export function canUseEventBridge(hostname = window.location.hostname): boolean {
   const bridge = getEventBridge()
-
-  return !!bridge && includes(bridge.getAllowedWebViewHosts(), window.location.hostname)
+  return (
+    !!bridge &&
+    bridge.getAllowedWebViewHosts().some((host) => {
+      const escapedHost = host.replace(/\./g, '\\$&')
+      const isDomainOrSubDomain = new RegExp(`(^|.+\\.)${escapedHost}$`)
```

Comment:
```suggestion
      const isDomainOrSubDomain = new RegExp(`^(.+\\.)*${escapedHost}$`)
```
personal preference, it seems more explicit to me that we can have 0 to n subdomains

Reply:
Why not